core/src/com/unciv/logic/civilization/ReligionManager.kt

```diff
@@ -204,10 +204,23 @@ class ReligionManager {
         civInfo.religionManager.foundingCityId = prophet.getTile().getCity()!!.id
     }
 
-    fun getBeliefsToChooseAtFounding(): BeliefContainer {
+    fun getBeliefsToChooseAtFounding(): Counter<BeliefType> {
+        val beliefsToChoose: Counter<BeliefType> = Counter()
+        beliefsToChoose.add(BeliefType.Founder, 1)
+        beliefsToChoose.add(BeliefType.Follower, 1)
         if (shouldChoosePantheonBelief)
-            return BeliefContainer(pantheonBeliefCount = 1, founderBeliefCount = 1, followerBeliefCount = 1)
-        return BeliefContainer(founderBeliefCount = 1, followerBeliefCount = 1)
+            beliefsToChoose.add(BeliefType.Pantheon, 1)
+            
+        for (unique in civInfo.getMatchingUniques("May choose [] additional [] beliefs when [] a religion")) {
```

Comment:
Please enum-ify this into "May choose [amount] additional [beliefType] beliefs when [foundingOrEnhancing] a religion"
That last param will be parsed as an "unknown param", which is absolutely fine! That's why it exists in the first place :)